hatchery/src/world.rs

```diff
@@ -193,6 +200,17 @@ impl World {
         Ok(id)
     }
 
+    /// Deploys a [`HostQuery`] with the given module ID.
+    pub fn deploy_host<Q>(&mut self, name: &'static str, query: Q)
```

Comment:
name could (should) be `Cow<'static, str>`, this would let deploy_host accept any kind of string with no penalty